.github/workflows/ci.yml

```diff
@@ -0,0 +1,36 @@
+name: CI
+
+on:
+  push:
+    # Run in PRs and for bors, but not on master.
+    branches:
+      - 'auto'
+      - 'try'
+      - '!gh-pages'
```

Comment:
Why is this needed?

Reply:
redundant when copied from other templates where every branches allowed but gh-page, will remove.